base_classes/NXxrayedge.nxdl.xml

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+  # NXreflections: candidate NeXus definition for X-ray edge spectroscopic and resonance data
+-->
+<definition type="group" name="NXxrayedge" category="base" extends="NXobject"
+            xmlns="http://definition.nexusformat.org/nxdl/3.1"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"  >
+  <symbols>
+    <symbol name="element"><doc>absorbing or resonant element</doc></symbol>
```

Comment:
We already have the fields `element` and `edge` with associated doc, so no need to define symbols with those names.

An example on how symbols are typically used
https://github.com/nexusformat/definitions/blob/d669ffb453ed5a89ca746f8d440adc1b9a5ecc05/base_classes/NXdetector.nxdl.xml#L44-L48

In our case this could be `Z: atomic number` for example.

Reply:
I am either completely lost or completely disagree with this comment.  Where is "element" used to mean "atomic element"?  I do not see "edge" anywhere.  

For sure, words like "element" and "channel" can take on multiple meanings.  Isn't that why there is a hierarchy?

And, no to using Z.   The atomic symbol should be used.